index.html

```diff
@@ -0,0 +1,285 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="./css/main.css" />
+  <link rel="icon" href="./favicon.ico" />
+  <title>Aziz Mejri - Portfolio</title>
+</head>
+
+<body>
+  <header class="header">
+    <div class="logo-menubars">
+      <h1 id="logo">Aziz Mejri</h1>
+      <div class="menu-btns">
+        <button type="button" class="menu-bars">
+          <i class="fas fa-bars"></i>
+        </button>
+        <button type="button" class="close-menubars">
+          <i class="fas fa-times"></i>
+        </button>
+      </div>
+    </div>
+    <nav class="desktop-nav">
+      <ul>
+        <li><a class="nav-link" href="#projects">Portoflio</a></li>
```

Comment:
I think the word that you were looking for was Portfolio but don't worry, is a minimal mistake 👍